core/app/models/spree/weighted_adjustments_handler.rb

```diff
@@ -0,0 +1,47 @@
+module Spree
+  class WeightedAdjustmentsHandler
```

Comment:
Would happily rename this class to something else. Additionally, only the calculator is currently under test. I intend to switch to placing this under test and just making sure that the calculator calls this with the correct arguments.